sparta/simdb/test/CoreDatabase/CMakeLists.txt

```diff
@@ -1,8 +1,7 @@
 project(SIMDB_CoreDatabase_test)
 
 add_executable(SIMDB_CoreDatabase_test CoreDatabase_test.cpp)
-
-include(../TestingMacros.cmake)
+include(${SPARTA_CMAKE_MACRO_PATH}/SimdbTestingMacros.cmake)
```

Comment:
Overall, this PR fixes quite a few things but I'm coming across this.  Putting `SimdbTestingMacros.cmake` in Sparta's `cmake/` subdirectory and defining `SPARTA_CMAKE_MACRO_PATH` in the toplevel `sparta/CmakeLists.txt` makes it so that simdb can't be built and regressed on it's own as a "cmake project".  @klingaard Was this intended?  Seems slightly odd if not broken to me.